# pygeocoder module is not working
df = pd.read_csv('geocodes.csv', encoding="iso-8859-1")
